src/clr-angular/data/datagrid/enums/render-step.enum.ts

```diff
@@ -11,4 +11,5 @@ export enum DatagridRenderStep {
   CLEAR_WIDTHS, // Note this is listened to by both cells and columns
   COMPUTE_COLUMN_WIDTHS,
   DETECT_STRICT_WIDTHS,
+  RESET,
```

Comment:
The more I read this, the more I think `RESET` and `CLEAR_WIDTHS` are exactly the same step. Not sure why we still have the two of them.

Reply:
I added it per your request 🥇 but will remove.